app/editor/src/features/content/list-view/components/tool-bar/filter/AdvancedSearchSection.tsx

```diff
@@ -43,9 +43,13 @@ export const AdvancedSearchSection: React.FC<IAdvancedSearchSectionProps> = () =
 
   const onChange = React.useCallback(() => {
     storeFilter({ ...filter, pageIndex: 0 });
-    storeFilterAdvanced({ ...filterAdvanced });
+    storeFilterAdvanced({
+      ...filterAdvanced,
+      startDate: oFilterAdvanced.startDate,
+      endDate: oFilterAdvanced.endDate,
+    });
     replaceQueryParams({ ...filter, pageIndex: 0, ...filterAdvanced }, { includeEmpty: false });
-  }, [filter, filterAdvanced, storeFilter, storeFilterAdvanced]);
+  }, [filter, oFilterAdvanced, filterAdvanced, storeFilter, storeFilterAdvanced]);
```

Comment:
This will cause an unwanted issue where the advanced filter will fire every time a key is pressed.  The advanced filter requires the user to click the search button.

Reply:
updated, thanks